index.js

```diff
@@ -1,9 +1,16 @@
 'use-strict';
 
-var xml = require('xml');
-var Base = require('mocha').reporters.Base;
-var filePath = process.env.MOCHA_FILE || 'test-results.xml';
-var fs = require('fs');
+var fs      = require('fs');
+var path    = require('path');
+var mkdirp  = require('mkdirp');
+
+var xml     = require('xml');
+var Base    = require('mocha').reporters.Base;
+
+function FILE_PATH() {
```

Comment:
I think it would be better to assign the result of `path.resolve` to a var like:
```javascript
var FILE_PATH = path.resolve(process.env.MOCHA_FILE || 'test-results.xml');
```
Seems like an unnecessary function call.